fn-fdk.js

```diff
@@ -8,6 +8,9 @@ let path = require('path')
 
 const fnFunctionExceptionMessage = 'Exception in function, consult logs for details'
 
+let fnLogframeName
```

Comment:
Not super keen on global variables here - could this be tied to the context?

Reply:
Fixed